go.mod

```diff
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/apache/incubator-yunikorn-scheduler-interface v0.0.0-20200327234544-149aaa3d8e48
-	github.com/golang/protobuf v1.3.5 // indirect
```

Comment:
could we avoid this change

Reply:
a `go mod tidy` adds it back again on my system so I think we need to follow that and remove this from the change